images/keda/main.tf

```diff
@@ -0,0 +1,76 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+locals {
+  c2p = {
+    "controller" = "keda",
+    "adapter"    = "keda-adapter",
+    "webhooks"   = "keda-admission-webhooks",
+  }
+  components = toset(keys(local.c2p))
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest" {
+  for_each = local.components
+  source   = "../../tflib/publisher"
+
+  target_repository = each.key == "controller" ? var.target_repository : "${var.target_repository}-${each.value}"
```

Comment:
ensures we keep the same tagging structure for the "main" controller (`keda` vs `keda-keda`)